apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/Google.Cloud.Spanner.Data.IntegrationTests.csproj

```diff
@@ -10,7 +10,7 @@
     <NoWarn>1701;1702;1705;4014</NoWarn>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netcoreapp1.0|AnyCPU'">
-    <DefineConstants>TRACE;DEBUG;NETCOREAPP1_0</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;NETCOREAPP1_0;NETCOREAPP1_0</DefineConstants>
```

Comment:
This looks unnecessary... you're defining the same constant twice.
(It should also be implicit anyway, I think - if the TFM is netcoreapp1.0, I'd expect NETCOREAPP1_0 to be defined.)
Basically, I'd *expect* you to be able to get rid of this line entirely, as TRACE and DEBUG should be implicit too.